src/store.js

```diff
@@ -86,7 +86,11 @@ export class Store {
     this._helper.on('result', onHelperResult.bind(this));
     this._helper.on('search', onHelperSearch.bind(this));
 
-    this._helper.getClient().addAlgoliaAgent(`vue-instantsearch ${version}`);
+    const client = this._helper.getClient();
+
+    if (typeof client.addAlgoliaAgent === 'function') {
+      client.addAlgoliaAgent(`vue-instantsearch ${version}`);
```

Comment:
I wonder if we could add this conditionally in the helper so we can call it without checking here

Reply:
I'm OK with this change. It's yet another helper upgrade though haha. WDYT @bobylito?